### ISSUE TITLE
def.js.Object.$set should return the Object

### ISSUE THREAD

**Issue by WhiteTrashLord** [cincheo/jsweet#497]
Would it be possible to change the return type of the method def.js.Object.$set so it returns the Object itself? Does it currently return the old value?

Code like this would be much shorter :)

.$set("key1", "value1").$set("key2", "value2")...

**Comment by renaudpawlak** [comment]
I just can't believe we have not thought of it before. Have you tried to change the prototype and see what happens with the generated code? There is a chance it works as is :)

**Comment by WhiteTrashLord** [comment]
I am using the Eclipse plugin. It's a bit hard for me to change the prototype. I hope you can test it yourself :)

**Comment by lgrignon** [comment]
@renaudpawlak doesn't it transpile to `obj["key1"] = value1`? If so, chaining won't work.

**Comment by renaudpawlak** [comment]
@lgrignon: yep you are right. My bad. It requires a change in the  core adapter :)
